stl/inc/xmemory

```diff
@@ -1057,7 +1071,8 @@ void _Deallocate_plain(_Alloc& _Al, typename _Alloc::value_type* const _Ptr) noe
 // FUNCTION TEMPLATE _Delete_plain_internal
 template <class _Alloc>
 _CONSTEXPR20_DYNALLOC void _Delete_plain_internal(_Alloc& _Al, typename _Alloc::value_type* const _Ptr) noexcept {
-    // destroy *_Ptr in place, then deallocate _Ptr using _Al; used for internal container types the user didn't name
+    // destroy *_Ptr in place, then deallocate _Ptr using _Al; used for internal container types the user didn't
+    // name

```

Comment:
Was this intentional, or the result of clang-formatting with too few `}` causing clang-format to indent the rest of the file, wrapping comments that overflow that won't be restored when the missing `}` is? (Yes, I do this - and fix the fallout - a lot.)

Reply:
It was, I thought I reverted all of those :(

Should be gone with the new commit